azure-mgmt-graph-rbac/src/main/java/com/microsoft/azure/management/graphrbac/ActiveDirectoryGroup.java

```diff
@@ -6,13 +6,15 @@
 
 package com.microsoft.azure.management.graphrbac;
 
+import com.microsoft.azure.management.apigeneration.LangDefinition;
 import com.microsoft.azure.management.graphrbac.implementation.ADGroupInner;
 import com.microsoft.azure.management.resources.fluentcore.model.Creatable;
 import com.microsoft.azure.management.resources.fluentcore.model.Wrapper;
 
 /**
  * An immutable client-side representation of an Azure AD group.
  */
+@LangDefinition
```

Comment:
Note we now have a simplified annotations approach: just use @Fluent() at the top interface and nothing else is needed. The embedded interfaces will be processed automatically - no need to annotate them (per Alva).
The LangDefinition imports can simply be all removed.